salt/utils/templates.py

```diff
@@ -345,11 +346,17 @@ def render_jinja_tmpl(tmplstr, context, tmplpath=None):
                                        **env_args)
 
     jinja_env.filters['strftime'] = salt.utils.date_format
-    jinja_env.filters['sequence'] = salt.utils.jinja.ensure_sequence_filter
     jinja_env.filters['yaml_dquote'] = salt.utils.yamlencoding.yaml_dquote
     jinja_env.filters['yaml_squote'] = salt.utils.yamlencoding.yaml_squote
     jinja_env.filters['yaml_encode'] = salt.utils.yamlencoding.yaml_encode
 
+    for function_name in context['salt'].keys():
+        # Simply iterating over the keys forces the execution modules to load
```

Comment:
It works for execution modules due to that loop. It works for utils modules either way. I'm hoping someone from the salt team, more familiar with the loader, can tell me if it is possible to get the same result without the loop.

The benefit to this implementation is that we can use the same approach to adding jinja filters across the salt code base, rather than having one approach for execution modules and another approach for utils modules.